### ISSUE TITLE
How does iRODSMetaCollection object have no get attribute?

### ISSUE THREAD

**Issue by tanmayagrawal21** [irods/python-irodsclient#634]
```
{"ansible_loop_var": "item", "changed": false, "item": "/testing/trash/home/shared", "msg": "An unexpected exception of type AttributeError occurred. Arguments: (\"'iRODSMetaCollection' object has no attribute 'get'\",)"}
```

**Comment by alanking** [comment]
`iRODSMetaCollection` does not appear to have a `get` attribute: https://github.com/irods/python-irodsclient/blob/e982023e77ac53ddda9b0b01026d08d564aa46fb/irods/meta.py#L77-L209 Are you looking for `get_all` or `get_one`?

**Comment by d-w-moore** [comment]
Note `get_all` retrieves all AVUs associated with a certain key (passed as the argument).
But, there's also the `items` method,  which does actually fetches **all** metadata associated with the object.  So you could say the get attribute is there, just ... misnamed.

**Comment by trel** [comment]
Cause for an alias?     Or would that make matters worse?

**Comment by d-w-moore** [comment]
> Cause for an alias? Or would that make matters worse?

I don't think it would hurt, and we've renamed a few things in the PRC already.

**Comment by korydraughn** [comment]
Is there an `iRODSMetaDataObject` class we can compare to? _I didn't see one, which seems a little weird._

I noticed `__getitem__` on `iRODSMetaCollection` which allows fetching info using the subscript operator.

@tanmayagrawal21 Where is it stated that the class supports `.get`? I didn't see it in the link you posted.
